build.sbt

```diff
@@ -21,5 +30,24 @@ lazy val core = project
   .settings(name := "quasar-destination-s3")
   .settings(
     performMavenCentralSync := false,
+    quasarPluginName := "s3-dest",
```

Comment:
Worth noting this is called `s3-dest` rather than `s3`. In order to avoid conflicts with the datasource.